Workflow/CIDER_Workflow.ipynb

```diff
@@ -433,13 +414,15 @@
      "output_type": "stream",
      "text": [
       "There are 0 duplicates in the first set.\n",
-      "SMILES strings for the duplicates: set()\n"
+      "SMILES strings for the duplicates: []\n",
+      "Index for the duplicates: []\n"
      ]
     }
    ],
    "source": [
     "print('There are ' + str(testdict[first_subdict]['number_of_duplicates']) + ' duplicates in the first set.')\n",
-    "print('SMILES strings for the duplicates: ' + str(testdict[first_subdict]['duplicates']) )"
+    "print('SMILES strings for the duplicates: ' + str(testdict[first_subdict]['duplicates_id']) )\n",
```

Comment:
Maybe change this to the phenole set where we actually have duplicates to print?
Also, could you add a comment here about the detected duplicates in the test sets, as discussed here https://github.com/hannbus/ChemIcal-DatasEt-comparatoR/issues/60 ? And "demo" the retrieval of the duplicate molecules from the mol supplier via the indices, maybe?

Reply:
The idea was that one should be able to use the notebook only with changing the imported data. Therefore, when calling the dictionary in the notebook, there is always the first subdictionary used (without its key) as this has to be there. Should I put this in nevertheless?

Reply:
We also have some other cells in the workflow notebook that only work with the example datasets, e.g. the retrieval of COCONUT ids, right? Maybe simply put try-except blocks around those so that they don't cause issues when executing the notebook with another dataset? Or can you check whether the example datasets are used or not and only execute those cells in the former case, using a conditional statement?

I think for these retrieval tasks, the most important point is that they are demoed with code examples in the notebook.